tests/conftest.py

```diff
@@ -445,12 +446,19 @@ def fixture_search_server():
 
 
 @pytest.fixture(name="setup_search_test")
-def fixture_setup_search_test(selenium):
+def fixture_setup_search_test(selenium, firefox):
     def _():
+        serp_import_uri = None
+
+        if firefox.browser.firefox_version >= 138:
+            serp_import_uri = "moz-src:///browser/components/search/SearchSERPTelemetry.sys.mjs"
+        else:
+            serp_import_uri = "resource:///modules/SearchSERPTelemetry.sys.mjs"
+
```

Comment:
I would suggest that we try/catch the import here.

Additionally there is no reason to use lazy because we're immediately using what we're importing.


```js

let SearchSERPTelemetry;
try {
    SearchSERPTelemetry = ChromeUtils.importESModule("moz-src:///browser/components/search/SearchSERPTelemetry.sys.mjs").SearchSERPTelemetry;
} catch (e) {
    SearchSERPTelemetry = ChromeUtils.importESModule("resource:///modules/SearchSERPTelemetry.sys.mjs").SERPSearchTelemetry;
}
```

Reply:
I can do that.